src/cascadia/TerminalApp/SettingsTab.cpp

```diff
@@ -0,0 +1,177 @@
+    namespace MUX = Microsoft::UI::Xaml;
+    namespace WUX = Windows::UI::Xaml;
+}
+
+namespace winrt::TerminalApp::implementation
+{
+    SettingsTab::SettingsTab()
+    {
+        Content(winrt::Microsoft::Terminal::Settings::Editor::MainPage());
+
+        _MakeTabViewItem();
+        _MakeSwitchToTabCommand();
+        _CreateContextMenu();
+        _CreateIcon();
+    }
+
+    // Method Description:
+    // - Initializes a TabViewItem for this Tab instance.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>
+    void SettingsTab::_MakeTabViewItem()
+    {
+        TabViewItem(::winrt::MUX::Controls::TabViewItem{});
+        TabViewItem().Header(winrt::box_value(Title()));
+    }
+
+    // Method Description:
+    // - Focus the settings UI
+    // Arguments:
+    // - focusState: The FocusState mode by which focus is to be obtained.
+    // Return Value:
+    // - <none>
+    void SettingsTab::Focus(WUX::FocusState focusState)
+    {
+        _focusState = focusState;
+
+        if (_focusState != FocusState::Unfocused)
+        {
+            Content().Focus(focusState);
+        }
+    }
+
+    WUX::FocusState SettingsTab::FocusState() const noexcept
+    {
+        return _focusState;
+    }
+
+    // Method Description:
+    // - Set the icon on the TabViewItem for this tab.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>
+    winrt::fire_and_forget SettingsTab::_CreateIcon()
+    {
+        auto weakThis{ get_weak() };
+
+        co_await winrt::resume_foreground(TabViewItem().Dispatcher());
+
+        if (auto tab{ weakThis.get() })
+        {
+            auto fontFamily = winrt::WUX::Media::FontFamily(L"Segoe MDL2 Assets");
+            auto glyph = L"\xE713"; // This is the Setting icon (looks like a gear)
+
+            // The TabViewItem Icon needs MUX while the IconSourceElement in the CommandPalette needs WUX...
+            IconSource(GetFontIcon<winrt::WUX::Controls::IconSource>(fontFamily, 12, glyph));
+            TabViewItem().IconSource(GetFontIcon<winrt::MUX::Controls::IconSource>(fontFamily, 12, glyph));
+
+            // Update SwitchToTab command's icon
+            SwitchToTabCommand().IconSource(GetFontIcon<winrt::WUX::Controls::IconSource>(fontFamily, 12, glyph));
+        }
+    }
+
+    // Method Description:
+    // - Gets the title string of the settings UI
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - the title string of the last focused terminal control in our tree.
+    winrt::hstring SettingsTab::GetActiveTitle() const
+    {
+        // TODO: This _could_ change depending on what page of the settings UI is open/focused?
+        return Title();
+    }
+
+    // Method Description:
+    // - Prepares this tab for being removed from the UI hierarchy
+    void SettingsTab::Shutdown()
+    {
+        // TODO: Does/Will the settings UI need some shutdown procedures?
+        Content(nullptr);
+        _ClosedHandlers(nullptr, nullptr);

```

Comment:
<details>
<summary>My train of thought 🚂🧠</summary>
When the Settings UI is opened, we to create a clone of CascadiaSettings.
As the user makes changes, we directly modify that clone.
When the user saves the changes, that clone gets promoted to be the real CascadiaSettings.
When we close the Settings UI, the Settings UI will own that clone. So the clone dies. All of the Settings UI is also destroyed, so I think that'll be fine.
</details>
I think we'll be fine. Would definitely like a second opinion from @DHowett if possible.